crates/router/src/connector/trustpay/transformers.rs

```diff
@@ -950,6 +950,7 @@ impl<F, T>
                     },
                 ))),
             }),
+            status: storage_models::enums::AttemptStatus::Pending,
```

Comment:
by default also it is Pending right?

Reply:
It's better to be explicit than rely on the caller. This flow defies the normal conventions that we follow.